DiscordBot.ActivityLogging/Helpers/LogSenderHelper.cs

```diff
@@ -0,0 +1,128 @@
+﻿using Discord.WebSocket;
+using Discord;
+using DiscordBot.ActivityLogging.Enums;
+using DiscordBot.ActivityLogging.Helpers.Models;
+using DiscordBot.Core.Helpers;
+using DiscordBot.Core.Interfaces;
+using Microsoft.Extensions.Logging;
+
+namespace DiscordBot.ActivityLogging.Helpers
+{
+    /// <summary>
+    /// Responsible for providing common log method
+    /// </summary>
+    public interface ILogSenderHelper
+    {
+        /// <summary>
+        /// Method that send activity logs on disord channel based on provided content
+        /// </summary>
+        /// <param name="content">Activity object that will be logged</param>
+        /// <returns>Task for discord SendMessageAsync()</returns>
```

Comment:
You can use <see> to refer to ther places in code like classes, interfaces, properties and methods.

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#see